devops/Makefile

```diff
@@ -25,6 +25,7 @@ export APPLICATION_ARCHIVE_DAYS := $(or ${APPLICATION_ARCHIVE_DAYS}, 43)
 export REDIS_PORT := $(or ${REDIS_PORT}, 6379)
 export API_PORT := $(or ${API_PORT}, 7070)
 export QUEUE_CONSUMERS_PORT := $(or ${QUEUE_CONSUMERS_PORT}, 7000)
+export WORKERS_PORT := $(or ${WORKERS_PORT}, 8000)
```

Comment:
Is there a reason to `WORKERS_PORT` to be different here?

Reply:
we usually used the port in the openshift different than the one we use locally, example QUEUE_CONSUMERS_PORT = 7000 in openshift and in local it is 3010. SO trying to follow the same here as 3020 and 8000.